cloud-sql/mysql/Startup.cs

```diff
@@ -55,152 +49,11 @@ public void ConfigureServices(IServiceCollection services)
                     options.Version = "Test";
                 });
             }
-            services.AddScoped(typeof(DbConnection), (IServiceProvider) =>
-                InitializeDatabase());
+            services.AddScoped(sp => new MySqlConnection().GetMySqlConnectionString());

```

Comment:
nit: This can now be a singleton. The connection string is only modified at startup so.